04_environment/index.tex

```diff
@@ -1 +1,275 @@
-\chapter{Game Environment}
\ No newline at end of file
+\chapter{Environment}
+\section{Foraging}
+\subsection{Foraging Background}
+
+Foraging is a fundamental requirement that is needed as it is a method of obtaining more resources from the starting amount. Foraging is based on the concept of inputting resources to get a return.
+
+\subsubsection{Underlying Concept}
+
+The primary objective of foraging in this game is to introduce a dilemma to the agents in order to investigate their behaviours and whether they work together despite different personal goals. Including two methods of foraging promotes decision making among the islands, allowing for strategic interactions to maximise personal benefits whilst still reaching the overarching goal of survival. 
+
+The inspiration for our concept of foraging comes from the stag hunt game furthermore it introduces elements from it. Specifically, this approach results in a game with \textbf{no dominant strategy} and in which \textbf{one agents choice can impact another agent}.
+
+The concept of stag hunt is that each agent selects a method of hunting without information about the others. The social welfare (total payoff) is significantly reduced if their choices differ. The selection of stag suggests that the agent is going for higher \textbf{payoffs} whilst the selection of hare indicates that the agent is going for \textbf{risk aversion} as both agents are required to stag hunt to get any personal welfare.
```

Comment:
Perhaps not _concept_ here? I think process is what you're writing about

Reply:
~ information about the others~ informing the others? the information of the other agents?

Reply:
~is going for \textbf{risk aversion}~ is risk averse?